src/Header.vue

```diff
@@ -37,20 +59,25 @@ export default {
     colors: {
       type: Object,
       required: true
-    }
-  },
-  data() {
-    return {
-      inUserList: false
+    },
+    showChatListButton: {
+      type: Boolean,
+      default: false
+    },
+    disableListToggle: {
+      type: Boolean,
+      required: true
     }
   },
   computed: {
-    ...mapState(['disableUserListToggle', 'titleImageUrl', 'showCloseButton'])
```

Comment:
We don't want user list toggle button appear in the header when chat list is shown.